lib/types.ts

```diff
@@ -21,6 +21,7 @@ export type SupportedFormat =
   | "icu";
 
 export interface ConfigYAML {
+  mode?: "ios";
```

Comment:
What is this for?

Reply:
oops, left behind from original implementation